services/movie.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models.query import QuerySet
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids is not None:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    if actors_ids is not None:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Are you sure that you return QuerySet when there get, has got only one argument?